requirements.txt

```diff
@@ -1,9 +1,9 @@
 cachetools>=2.0,<2.1
 certifi>=2017
-pendulum>=1.2.3,<1.3
+pendulum==1.2.5
```

Comment:
the latest pendulum is 2.1.2we want to upgrade to the latest, locking swimlane to old doesn't help when other libraries install latest one. That is how we got into problems.

Reply:
updated